crates/weaver_live_check/README.md

```diff
@@ -0,0 +1,239 @@
+# Weaver Live Check
+
+Live check is a developer tool for assessing sample telemetry and providing advice for improvement.
+
+A Semantic Convention `Registry` is loaded for comparison with samples. `Ingesters` transform various input formats and sources into intermediary representations to be assessed by `Advisors`. The `Advice` produced is transformed via jinja templates to the required output format for downstream consumption.
+
+```mermaid
+flowchart LR
+    subgraph Inputs
+        file["File"]
+        stdin["stdin"]
+        otlp["OTLP"]
+    end
+
+    subgraph Core["Processing"]
+        registry["Registry"]
+        ingesters["Ingesters"]
+
+        subgraph advisors["Advisors"]
+            builtin["Builtin"]
+
+            subgraph external["External"]
+                otel["Otel"]
+                custom["Custom"]
+            end
+        end
+    end
+
+    subgraph Outputs
+        advice["Advice"]
+        templates["Jinja Templates"]
+        output["Output Format"]
+    end
+
+    file --> ingesters
+    stdin --> ingesters
+    otlp --> ingesters
+
+    registry -- "Loaded for comparison" --> advisors
+    ingesters -- "Intermediary representations" --> advisors
+
+    builtin --> advice
+    external --> advice
+
+    advice -- "Transformed via" --> templates
+    templates --> output
+```
+
+## Ingesters
+
+Sample data can have various levels of detail; from a simple list of attribute names, to a full OTLP signal structure. This data can come from different sources: files, stdin, OTLP. Therefore you need to choose the appropriate `Ingester` for your job:
+
+- `attribute_file | AF | af` => for attribute name or name=value pair text files
```

Comment:
This seems to put a huge focus on "attributes" vs. "signals".  I'm not positive that's the right focus we should have for Weaver.

Yes, semantic conventions has a huge focus on attributes, by we really want to push for *signals* and signal generation code (i.e. span, metric, event and resource[entity]s).

While I think it's fine if the PR only supports attributes initially, I think we should think carefully about ingesters we recommend.

I'd rather see `{signal}_json_file`, `{signal}_otlp` and `{signal}_json_stdin` where we only pay attention to attributes initially.

Reply:
I have a few commits coming in that change how `Ingesters` are configured on the CLI. 

TBH I just did attributes in this first PR because I was trying to limit scope but show off lots of use-cases to whet the appetite. I envisage the defaults flipping in future PRs.